Bio/Phylo/PhyloXML.py

```diff
@@ -668,27 +683,45 @@ def __floordiv__(self, other):
         return self.value.__floordiv__(other)
 
     def __rfloordiv__(self, other):
+        """Conduct revers C-style and old-style division."""
         return other.__floordiv__(self.value)
 
     def __mod__(self, other):
-        """Conduct modulus between value of two Confidence objects."""
+        """Conduct modulus between values of two Confidence objects."""
         return self.value % other
 
     def __rmod__(self, other):
+        """Conduct reverse modulus between values of two Confidence objects."""
         return other % self.value
 
     def __divmod__(self, other):
+        """Return quotient and remainder.
+
+        of dividing the values of two Confidence objects.
+        """
         return divmod(self.value, other)
 
     def __rdivmod__(self, other):
+        """Return quotient and remainder.
+
+        of dividing and reversing the values of two Confidence objects.
+        """
         return divmod(other, self.value)
 
     def __pow__(self, other, modulo=None):
+        """Return the value of Confident object raised at power of.
```

Comment:
This does not make sense (unless you include the rest of the docstring). How about:

```
"""Return value of confidence object raised to the given power."""

Also, typo: Confident object.